tests/test_bootstrap_cli.py

```diff
@@ -389,3 +384,9 @@ def setup_newenv_environment(tmp_path, runner):
     os.mkdir(f"{tmp_path}/copilot/environments/newenv")
     to_file = Path(tmp_path / "copilot/environments/newenv/manifest.yml")
     shutil.copy(my_file, to_file)
+
+
+def switch_to_tmp_dir_and_copy_config_file(tmp_path, valid_config_file):
+    os.chdir(tmp_path)
+    Path("bootstrap.yml").parent.mkdir(parents=True, exist_ok=True)
```

Comment:
What is this line doing?

Reply:
No longer needed. I was using it to create parent dirs when I'd repurposed it to copy arbitrary files. No longer needed though.